resource-managers/kubernetes/docker/src/main/spark-base/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Think we want `src/main/dockerfiles` as the top level directory.

Reply:
Done.